### DISCUSSION TITLE
v0.7

### DISCUSSION THREAD

**Discussion by ililim** [ililim/dual-key-remap#45] [Announcements]
Code Diff: https://github.com/ililim/dual-key-remap/compare/v0.6...master

### Fixed
- Dual-key-remap will no longer show a blank console window when launched via the Windows Terminal app. This issue mainly affected Win11 users due to Terminal replacing Command Prompt as the default console application.
- A console window will still show up if you specify `DEBUG=1` in your config.

<hr /><em>This discussion was created from the release <a href='https://github.com/ililim/dual-key-remap/releases/tag/v0.7'>v0.7</a>.</em>

**Comment by imakahn** [comment]
This is blocked by windows defender as having the `Trojan:Win32/Wacatac.B!ml` trojan. @ililim

**Reply by ililim** [reply]
Thanks for letting me know! I've submitted it as a false positive to https://www.microsoft.com/en-us/wdsi/filesubmission. For anyone who wants to us this in the mean time feel free to use v0.6 which I believe isn't flagged.

**Reply by ililim** [reply]
Tracking this in https://github.com/ililim/dual-key-remap/issues/46, check back there for updates.

**Reply by ililim** [reply]
OK! Should be resolved :)

**Reply by jens120** [reply]
Windows defender today has identified dual-key-remap.exe v0.7 as a low threat: `PUA:Win32/Packunwan`. @ililim 

Windows Defender says I have: Antivirus Version: 1.415.127.0

**Reply by dumblob** [reply]
> Windows defender today has identified dual-key-remap.exe v0.7 as a low threat: `PUA:Win32/Packunwan`. @ililim
> 
> Windows Defender says I have: Antivirus Version: 1.415.127.0

I see the same issue (Antivirus Version: 1.417.391.0).

**Comment by imakahn** [comment]
Thank you! This util makes windows usable for me

**Reply by ililim** [reply]
Glad to hear people find it useful :)
